setup.cfg

```diff
@@ -1,2 +1,5 @@
 [metadata]
 description-file = README.rst
+
+[wheel]
+universal = 1
```

Comment:
I have to read up what exactly this does, but from a quick Google search it seems to be only relevant for packages that support Python 2 and 3?

Reply:
Might be true.  It can be removed than. :)

Reply:
I think so indeed: https://packaging.python.org/guides/distributing-packages-using-setuptools/#universal-wheels
Also, it seems the proper name for the section is `[bdist_wheel]`.

Reply:
I think we can get rid of the complete `setup.cfg` file.  The long description is already in `setup.py`.